### ISSUE TITLE
Is it possible to support Texture Healing?

### ISSUE THREAD

**Issue by yvvt0379** [JetBrains/JetBrainsMono#653]
[Monaspace](https://github.com/githubnext/monaspace) supports Texture Healing. It shifts the width of narrow letters (e.g. `i` and `l`) and wide letters (e.g. `m` and `w`) to make the text more balanced in monospaced context. Is it possible for JetBrains Mono to support Texture Healing?

![image](https://github.com/JetBrains/JetBrainsMono/assets/62426149/aedfeb7f-2ebf-4e76-b5df-94f2987ff633)

**Comment by mon-jai** [comment]
May be the current state is good enough?

Top: JetBrains Mono
Bottom: Monaspace with Texture Healing enabled

![image](https://github.com/JetBrains/JetBrainsMono/assets/91261297/fc2495e3-78e2-46e9-9b9f-574c6cc7b24d)

**Comment by tianfanghan** [comment]
Yes this is a very good idea for jetbrains mono update, please think about it designers.

**Comment by tianfanghan** [comment]
**Please notice this, thank you developers.**
